### DISCUSSION TITLE
spglib-1.5.2/src/.libs/libsymspg.a files not found

### DISCUSSION THREAD

**Discussion by Lanlin123** [band-unfolding/bandup#40] [Q&A]
Dear Bandup developers,
 
When I build Bandup, I meet the error " ifort: error #10236: File not found:  '/bandup-master/src/external/spglib-1.5.2/src/.libs/libsymspg.a'. Can you give me some advice? Thank you.

Best regard,
Lanlin.

**Comment by stepan-tsirkin** [comment]
@Lanlin123 ,

my advise is to switch to banduppy : https://github.com/band-unfolding/banduppy

BandUp is very old, and we do not support it anymore.

If you want to proceed with Banduppy

> '/bandup-master/src/external/spglib-1.5.2/src/.libs/libsymspg.a'. 

regarding this, looks like a folder '/bandup-master' in the root directory is requested, which most probably does not exist. Use relative path, or complete the absolute path.
